src/controllers/system_prompt.py

```diff
@@ -0,0 +1,29 @@
+import os
+import dotenv
+from src.constants import (
+    DEFAULT_SYSTEM_PROMPT,
+    SYSTEM_PROMPT_WITH_TOOLS_PROMPT,
+    ENV_FILE_PATH,
+)
+from string import Template
+
+
+def update_system_prompts():
+    """
+    We construct the system prompt from the custom prompt and tool custom prompts.
+    Call this function to update the system prompt once the custom prompts and tool system prompts are updated.
+    """
+    # Initialize the system prompt from custom prompt or default prompt
+    system_prompt = os.getenv("CUSTOM_PROMPT", DEFAULT_SYSTEM_PROMPT)
+
+    tool_custom_prompts = os.getenv("TOOL_CUSTOM_PROMPTS", None)
```

Comment:
why we don't iterate over the tools config here instead (and don't persist `TOOL_CUSTOM_PROMPTS`)?

Reply:
great idea, that would be good! let me update it